MovieApp/PresentationLayer/Models/CreditsViewModel.swift

```diff
@@ -0,0 +1,19 @@
+struct CreditsViewModel {
+
+    let cast: [CastMemberViewModel]
+    let crew: [CrewMemberViewModel]
+
+}
+
+// MARK: ClientModel to DataSourceModel Conversion
+extension CreditsViewModel {
+
+    init(from model: CreditsModel) {
+        cast = model.cast.map { CastMemberViewModel(from: $0) }
+        crew = model.crew
+            .map { CrewMemberViewModel(from: $0) }
+            .sorted { $0.popularity > $1.popularity }
+            .dropLast(model.crew.count - 6)
```

Comment:
This is a bit confusing.... why not:
```
model.crew
            .map { CrewMemberViewModel(from: $0) }
            .sorted { $0.popularity < $1.popularity }
            .first(6)
```

But check if .first will throw an exception if the list count is less than 6.
Also, this sort of business logic should not be in init. Ideally, you should put it in use case or presenter layer, but not in init of a struct.